core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -259,9 +263,9 @@ var SkillEditorPage = function() {
 
     await (await browser.switchTo().activeElement()).sendKeys(explanation);
 
-    await waitFor.elementToBeClickable(
+    await waitFor.visibilityOf(
       saveConceptCardExplanationButton,
-      'Save Concept Card Explanation button takes too long to be clickable');
+      'Save Concept Card Explanation button takes too long to be appear');
```

Comment:
Since we click `saveConceptCardExplanationButton`, I think we do want to wait for it to be clickable. You could use `action.click`, which will wait for both visibility and clickability. Here's its implementation:

https://github.com/oppia/oppia/blob/96c404b45e61e53d07ca0e4f308b6d330bd49f81/core/tests/protractor_utils/action.js#L35

Reply:
done